src/converters/basic_sensors/moving.ts

```
@@ -0,0 +1,30 @@
+
+export class MovingSensorHandler extends BinarySensorHandler {
+  public static readonly exposesName: string = 'moving';
+
+  constructor(expose: ExposesEntryWithProperty, otherExposes: ExposesEntryWithBinaryProperty[], accessory: BasicAccessory) {
+    super(
+      accessory,
+      expose as ExposesEntryWithBinaryProperty,
+      otherExposes,
+      MovingSensorHandler.generateIdentifier,
+      'Motion Sensor (moving)',
+      (n, t) => new hap.Service.MotionSensor(n, (MovingSensorHandler.exposesName + ' ' + (t ?? '')).trim()),
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)